content/c-sharp/concepts/arrays/terms/clear/clear.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: '.Clear()'
+Description: 'Clears the contents of an array, returning each element to its default value.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Arrays'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The `.Clear()` method is used to clear the contents of an array, returning each element to its default value.
+
+## Syntax
+
+```pseudo
```

Comment:
```suggestion
```cs
```

Reply:
This looks like it should be a block of C# code.

Reply:
@HishamT for syntax sections we use pseudo blocks.